source/index-management/index-management.rst

```diff
@@ -852,6 +852,186 @@ Example::
 	> a
 	[ "_id_", "ty_1", "l_2dsphere", "ts_1" ]
 
+--------------
+Search Indexes
+--------------
+
+Server 7.0 introduced three new server commands and a new aggregation stage to facilitate management of search indexes.  Drivers MUST provide 
+an API similar to the existing index management API specifically for search indexes.  Drivers MAY choose to implement either the standard
+API or the index view API.
+
+Search Index Management Helper Options
+--------------------------------------
+
+There are currently no supported options for any of the search index management commands.  To future proof
+drivers implementations so that any options added in the future do not constitute a breaking change to drivers,
+empty options structs have been added as placeholders.  If a driver's language has a mechanism to add options 
+in a non-breaking manner (i.e., method overloading) drivers MAY omit the empty options structs from their
+search index management helpers.
+
+``listSearchIndexes`` is implemented using an aggregation pipeline.  The list helper MUST support a driver's aggregation
+options as outline in the `crud specification <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_.  Drivers MAY combine the aggregation options with
+any future ``listSearchIndexes`` stage options, if that is idiomatic for a driver's language.
+
+Notes
+-----
+
+The search index commands are asynchronous and return from the server before the index is successfully updated, created or dropped.
+In order to determine when an index has been created / updated, users are expected to run the ``listSearchIndexes`` repeatedly
+until index changes appear.
+
+An example, from Javascript:
+
+.. code:: typescript
+
+  const name = await collection.createSearchIndex({ definition: { ... fill out definition } })
+  while (!(await collection.listSearchIndexes({ name }).hasNext())) {
+    await setTimeout(1000);
+  }
+ 
+Common Interfaces
+-----------------
+
+.. code:: typescript
+
+  interface SearchIndexModel {
+    // Returns the definition for this index.
+    definition: Document;
+
+    // Returns the name for this index, if present.
+    name: Optional<string>;
+  }
+
+  /**
+   * The following interfaces are empty but are provided as placeholders for drivers that cannot 
+   * add options in a non-breaking manner, if options are added in the future.
```

Comment:
Is `Comment` supported on the new search commands? Has adding `Comment` to the options been considered?

Reply:
Comment is not supported (just the first comment in [this thread](https://mongodb.slack.com/archives/C04TMC263M3/p1683836146256049) for context).